wdl/MainVcfQc.wdl

```diff
@@ -858,3 +886,111 @@ task SanitizeOutputs {
   }
 }
 
+
+# Identify all duplicates in a single file
+task IdentifyDuplicates {
+  input {
+    String prefix
+    File vcf
+    String sv_pipeline_qc_docker
+    File? custom_script
+    RuntimeAttr? runtime_attr_override
+  }
+
+  # File default_script = "/src/sv-pipeline/scripts/merge_duplicates.py"
+  # File active_script = select_first([custom_script, default_script])
```

Comment:
Should re-enable the docker script

Reply:
Updated accordingly - removed the custom script.